### DISCUSSION TITLE
Status of GitHub wiki

### DISCUSSION THREAD

**Discussion by danieldegroot2** [openstreetmap/iD#9858] [General]
https://github.com/openstreetmap/iD/wiki

How far developed is the wiki? what is still needed?

**Comment by 1ec5** [comment]
“[Meetings](https://github.com/openstreetmap/iD/wiki/Meetings)” is definitely obsolete but probably shouldn’t be deleted, since it contains some information that could be of historical value. It looks like @tyrasd made an effort to [document localizer](https://github.com/openstreetmap/iD/wiki/How-to-use-iD%27s-localizer-module). Task-based documentation like this can be very helpful to newcomers, but I don’t know if it’s discoverable enough. The [contributing guide](https://github.com/openstreetmap/iD/blob/b792a0106daf844af75373ef1b85176cecbe13f9/CONTRIBUTING.md) doesn’t link to any wiki pages.

**Comment by danieldegroot2** [comment]
As of late 2025, there was some vandalism to the wiki. (spam additions and possibly reverts of valid changes by a second user)

**Reply by k-yle** [reply]
I see Minh already cleaned up [the vandalism](https://github.com/openstreetmap/iD/wiki/Home/_compare/6802d21e59e828f68ae312943c7d96f63a747e82...a4b5a90e1fce337b712ced586c8c4a0110ac9f36), thanks for that. 

Maybe the technical parts should be moved into [ARCHITECTURE.md](https://github.com/openstreetmap/iD/blob/develop/ARCHITECTURE.md) and the remainder to the OSM Wiki? (which already has pages that duplicate topics in the github wiki)
